proxies/java/src/main/java/com/dvc_harness/proxy/data/DataStore.java

```diff
@@ -0,0 +1,23 @@
+package com.dvc_harness.proxy.data;
+
+import java.util.Hashtable;
+
+import com.devcycle.sdk.server.cloud.api.DVCCloudClient;
+import com.devcycle.sdk.server.local.api.DVCLocalClient;
+import com.devcycle.sdk.server.common.model.*;
+
+class DataStore {
+    public static Hashtable<String, DVCLocalClient> LocalClients;
+    public static Hashtable<String, DVCCloudClient> CloudClients;
+
+    public static Hashtable<String, User> Users;
+    public static Hashtable<String, String> Commands;
+
+    public static void DataStore()
+    {
+        Hashtable CloudClients = new Hashtable<String, DVCCloudClient>();
+        Hashtable LocalClients = new Hashtable<String, DVCLocalClient>();
+        Hashtable Users = new Hashtable<String, User>();
+        Hashtable Commands = new Hashtable<String, Object>();
```

Comment:
this disagrees with the type definition of the `Commands` declaration above?

Reply:
also are you sure these are talking about the same property? You shouldn't need to re-declare the type being `Hashtable` for these assignments

Reply:
hmmm I really need to enable autosave in vscode so I stop committing incomplete changes

Reply:
ah I see what you mean